datasets/datasets.json

```diff
@@ -336,6 +336,18 @@
       "b": "keyword"
     }
   },
+  {
+    "name": "random-768-100-tenants",
+    "vector_size": 768,
+    "distance": "cosine",
+    "type": "tar",
+    "link": "https://storage.googleapis.com/ann-filtered-benchmark/datasets/random_keywords_1m_768_vocab_100.tgz",
+    "path": "random-768-100-tenants/random_keywords_1m_768_vocab_100",
+    "schema": {
+      "a": "keyword"
+    },
+    "tenants": [ "a" ]
```

Comment:
```suggestion
    "tenant_fields": [ "a" ]
```

Reply:
We can avoid adding `tenants` info in dataset. see my [comment](https://github.com/qdrant/vector-db-benchmark/pull/183/files#r1718153997)